frontends/mrxl/test/runt.toml

```diff
@@ -0,0 +1,38 @@
+ver = "0.4.0"
+
+[[tests]]
+name = "interpret"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {} --data {}.data --interpret"
+expect_dir = "expect/interpret_out/"
+
+[[tests]]
+name = "gen_futil"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {}"
+expect_dir = "expect/futil_out/"
+
+[[tests]]
+name = "convert_data"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {} --data {}.data --convert"
+expect_dir = "expect/data_out/"
+
+[[tests]]
+name = "simulate_interp"
+paths = [ "./*.mrxl" ]
+cmd = "fud e --from mrxl {} --to interpreter-out -s mrxl.data {}.data -q"
+expect_dir = "expect/simulate_interp_out/"
+
+[[tests]]
+name = "simulate_iverilog"
+paths = [ "./*.mrxl" ]
+cmd = "fud e --from mrxl {} --to dat --through icarus-verilog -s mrxl.data {}.data -q"
+expect_dir = "expect/simulate_iverilog_out/"
+
+[[tests]]
+name = "simulate_verilog"
+paths = [ "./*.mrxl" ]
+cmd = "fud e --from mrxl {} --to dat --through verilog -s mrxl.data {}.data -q"
+expect_dir = "expect/simulate_verilog_out/"
```

Comment:
It would probably be a good idea to share the expected output from these three test treatments. That is, we want Icarus, Verilator, and the interpreter to all agree.

AFAICT, the files for Verilator and Icarus are already identical. The interpreter uses a slightly different output format, but we have dealt with this difference elsewhere using a `jq` trick to pull out the memory contents only from all formats (which are supposed to be identical). Like this for Icarus and Verilator:
https://github.com/cucapra/calyx/blob/d03d22ffb3f5b13d3091e2d3e82d8fb3d0963f54/runt.toml#L122

Note also the `--to jq --through dat` trick to make fud first produce a data file and then `jq`-transform it:
https://github.com/cucapra/calyx/blob/d03d22ffb3f5b13d3091e2d3e82d8fb3d0963f54/runt.toml#L115-L117

And similarly for Cider:
https://github.com/cucapra/calyx/blob/d03d22ffb3f5b13d3091e2d3e82d8fb3d0963f54/interp/runt.toml#L102-L103

Seems like this could simplify things a bit!